src/main/java/seedu/address/model/datetime/Date.java

```diff
@@ -38,6 +38,16 @@ public String getValue() {
         return value;
     }
 
+    /**
+     * Returns the LocalDate of String value.
+     *
+     * @return LocalDate of the date.
+     */
+    public LocalDate getLocalDateValue() {
```

Comment:
It feels a bit strange that we made a separate Date class instead of just using LocalDate from the start, but it is what it is. (this is not a complaint of your PR)